rest/accounts/list-get-example-1/list-get-example-1.5.x.rb

```diff
@@ -0,0 +1,15 @@
+# Get twilio-ruby from twilio.com/docs/ruby/install
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# Initialize Twilio client
+@client = Twilio::REST::Client.new(account_sid, auth_token)
+
+
+# Loop over accounts and print out a property for each one
+@client.accounts.list.each do |account|:
```

Comment:
+1 on using the list resource directly. For the instance resources we will have to do massive replacements once the library is fixed